src/CSharp/AspnetWebMvc/AspnetWebMvc/Web.config

```diff
@@ -1,41 +1,43 @@
 <configuration>
   <appSettings>
-    <add key="webpages:Version" value="3.0.0.0"/>
-    <add key="webpages:Enabled" value="false"/>
-    <add key="ClientValidationEnabled" value="true"/>
-    <add key="UnobtrusiveJavaScriptEnabled" value="true"/>
-    <add key="IdentifyOauth2:Authority" value="https://dev55.safewhere.local/runtime/oauth2"/>    
+    <add key="webpages:Version" value="3.0.0.0" />
+    <add key="webpages:Enabled" value="false" />
+    <add key="ClientValidationEnabled" value="true" />
+    <add key="UnobtrusiveJavaScriptEnabled" value="true" />
+    <add key="IdentifyOauth2:Authority" value="https://dev55.safewhere.local/runtime/oauth2" />    
     <!--Already supported: client_secret_jwt, client_secret_basic, client_secret_post-->
-    <add key="IdentifyOauth2:UsingRequestObject" value="true"/>
-    <add key="IdentifyOauth2:SignRequestObject" value="false"/>
-    <add key="IdentifyOauth2:AuthenticationType" value="client_secret_jwt"/>
-    <add key="IdentifyOauth2:ClientSecret" value="webmvc_secret"/>
-    <add key="IdentifyOauth2:ClientCertificate" value="3C1FD735A4035E3B78D33444DE5327C393AA282E"/>
+    <add key="IdentifyOauth2:UsingRequestObject" value="true" />
+    <add key="IdentifyOauth2:SignRequestObject" value="false" />
+    <add key="IdentifyOauth2:AuthenticationType" value="client_secret_jwt" />
+    <add key="IdentifyOauth2:ClientSecret" value="webmvc_secret" />
+    <add key="IdentifyOauth2:ClientCertificate" value="3C1FD735A4035E3B78D33444DE5327C393AA282E" />
     <add key="IdentifyOauth2:IdTokenHint" value="eyJhbGciOiJSUzI1NiIsImtpZCI6IlBCX1hOYVFEWGp0NDB6UkUzbE1udzVPcUtDNCIsIng1dCI6IlBCX1hOYVFEWGp0NDB6UkUzbE1udzVPcUtDNCIsInR5cCI6IkpXVCJ9.eyJzdWIiOiJDZkRKOEFNTF9Vbi03czFMcWdUU3VMN1daYVpXRnpfVHNBd0lrTjNkMEVybWp1UDZsYlZRQWZzSllsc2ZseGRmeEMyT09nejRpeDJkZ3BVQXVCeVN4NExBUzRHT3ZnMXBfOE5uZy1aY0JrOFp1REEyU2YtbUx5U2xjM3dkVS16M1prbmNhMnhobmVKdTlQQlFzQktaY3dVUFBmeTNkWWthRVJZY0JHQnpYVVh2b2kxOSIsInVpSWQiOiJmN2I3MmNhMC03MzBjLTQxYTEtYTcyYy1hZTJmMGM5YzI4MTIiLCJhdXRoX3RpbWUiOjE1Njc5MTUxMjkuMCwidG9rZW5fdXNhZ2UiOiJpZGVudGl0eV90b2tlbiIsImp0aSI6ImU5OGZkMzI2LWFjYWMtNDhmOS1iMDEyLTdlODc2OWIwZDcxNCIsImF1ZCI6WyJ3ZWJtdmNfY29kZWZsb3dfaWQiLCJodHRwczovL2RldjU1LnNhZmV3aGVyZS5sb2NhbC9ydW50aW1lLyJdLCJub25jZSI6IjhjYWE1MTAzMmQwMzQzMjg4ODlhZjFkY2EwODViNzU5IiwiYXRfaGFzaCI6IlRodFFWUWFXMFgwcUlfdDBvTDlqVHciLCJhenAiOiJ3ZWJtdmNfY29kZWZsb3dfaWQiLCJpYXQiOjE1Njc5MTUxNDEsIm5iZiI6MTU2NzkxNTE0MSwiZXhwIjoxNjAzOTE1MTQxLCJpc3MiOiJodHRwczovL2RldjU1LnNhZmV3aGVyZS5sb2NhbC9ydW50aW1lL29hdXRoMiJ9.SPQsSscFBYPoXsxF2RbhlU9bezXhKa8XX8z6rEJtwkaQpqSYrhrlPW2Gu6ZdRfmNvRWa_tJRmDJXtEw6Eppy7Jz_iUwhnQ8CsZymNeGYS6LuzLlMK29uyk4Ta9w0hNDgHciR6X9FEQuJERVZSRKaYq9kl_qpmWG3nl6G0KPVLTa7aeBXVn6qyWjfz7WebH2Bw3P_BKtaSj1dKqXHT2AJn8EG6m1Q-dPOWw1j2XyxsQYQarSeNOuPZ9_A170hsGNCn1HERKeQ-hdrEKQTnFctdtbVD7srF96l4H6ZS-NLN772AdZwtgruRTvSM8EH7Uv4p2OCKcs6FhZi-fBQeoEtRA"/>
+    <add key="IdentifyOauth2:CodeVerifier" value="nX8nXTvRo5pAA1GedqTYF7yLykImlbUs9gEnkm-6ixI" />  <!--URL safe unique identifier-->
```

Comment:
This value should be auto-generated inside the source code. Configuration setting added to enable/disable it only

Reply:
Yes, I remove the setting for "code verifier" and generate it randomly for each session.